scripts/docker-entrypoint.sh

```diff
@@ -34,8 +37,8 @@ prconfig="${config_root}/prconfig.xml"
 context_xml="${config_root}/context.xml"
 tomcatusers_xml="${config_root}/tomcat-users.xml"
 
-db_username_file="${secret_root}/DB_USERNAME"
-db_password_file="${secret_root}/DB_PASSWORD"
+db_username_file=$(resolveSecretPath "${secret_root}" "$DB_USERNAME_SECRET_PATH" "DB_USERNAME")
+db_password_file=$(resolveSecretPath "${secret_root}" "$DB_PASSWORD_SECRET_PATH" "DB_PASSWORD")
 
 cassandra_username_file="${secret_root}/CASSANDRA_USERNAME"
```

Comment:
Are we holding off on allowing the non-DB secrets to be overridden?

Reply:
Oh as in put the resolveSecretPath for all of them? Yeah that's probably a good idea